tests/cleo_tests/.Compile_All.bat

```diff
@@ -0,0 +1,19 @@
+@REM Compile all .txt files in the current directory to .s files using Sanny Builder 4
+@REM Usage: set SANNY="path\to\sanny.exe" && .Compile_All.bat
+
+@echo off
+
+@REM Delete all .s files in the current directory and subdirectories
+for /f "delims=" %%i in ('dir /b /s *.s') do (
+    echo Deleting "%%i"...
+    del "%%i"
+)
+
+@REM Compile all .txt files in the current directory and subdirectories
+for /f "delims=" %%i in ('dir /b /s *.txt') do (
+    echo Compiling "%%i"...
+    %SANNY% --compile "%%i" "%%~pi%%~ni.s" --no-splash --mode sa_sbl
```

Comment:
`%%~pi` returns path without drive letter! Use `%%~dpi` instead.

Reply:
I also added printing just relative paths in window so it is not so crowded with repeating information.

Reply:
If there is compilation script then all compiled files can be removed from the repo.

Reply:
makes sense 👍